sklearn/model_selection/_search.py

```diff
@@ -704,19 +757,51 @@ class GridSearchCV(BaseSearchCV):
            fit_params={}, iid=..., n_jobs=1,
            param_grid=..., pre_dispatch=..., refit=...,
            scoring=..., verbose=...)
-
+    >>> sorted(clf.results_.keys())
+    ...                             # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+    ['param_C', 'param_kernel', 'params', 'test_mean_score',...
+     'test_rank_score', 'test_split0_score', 'test_split1_score',...
+     'test_split2_score', 'test_std_score']
 
     Attributes
     ----------
-    grid_scores_ : list of named tuples
-        Contains scores for all parameter combinations in param_grid.
-        Each entry corresponds to one parameter setting.
-        Each named tuple has the attributes:
-
-            * ``parameters``, a dict of parameter settings
-            * ``mean_validation_score``, the mean score over the
-              cross-validation folds
-            * ``cv_validation_scores``, the list of scores for each fold
+    results_ : dict of numpy (masked) ndarrays
+        A dict with keys as column headers and values as columns, that can be
+        imported into a pandas DataFrame.
+
+        For instance the below given table
+
+        param_kernel|param_gamma|param_degree|test_split0_score|...|...rank...|
```

Comment:
Is this table showing up in your sphinx build? For me, in the latest version of sphinx, everything from here to the end of the docstring disappears in both HTML and LaTeX builds. Might be a numpydoc bug?